src/stylesheets/settings/_mixins.scss

```diff
@@ -147,6 +147,7 @@
 @mixin button($color, $background-color, $background-hover, $border-color:transparent) {
 
   @include btn;
+  @include btn--toggle($color);
```

Comment:
This does imply that all buttons are a toggle, would cause confusion longterm. Better to add `content: ""` to the `btn--toggle` mixin and apply to a `button` within a `dropdown`

Reply:
I've removed the btn--toggle mixin completely and have put it in as styles inside dropdown and dropdown--block. However, to make the color for the caret the same as the font color for each button, I've added a border-top-color property inside the button mixin for the ::after pseudo class.